### PR TITLE
Add redoc debug setting

### PR DESCRIPTION
**Pull request by JonasNiestroj**
# Checklist (Only for frontend changes)
- [ ] Changes tested in IE11
- [ ] Responsive
- [ ] Documented "How to use it"
- [ ] Documented "How to install"
- [ ] Featured in Styleguide

# Summary of this PR

### CODE REVIEW

File: backend/app/settings.py [hunk 1]
```diff
@@ -177,3 +177,20 @@
...
+            'in': 'header',
+            'description': 'The api uses JSON Web Tokens to grant access to the '
+                           'endpoints which need permissions. Check the '
+                           'JSON Web Token section for more information.',
+            'name': 'Authorization',
+            'type': 'apiKey',
+        },
+    }
+}
+ENABLE_REDOC_ONLY_IN_DEBUG = True
```
Comment (21adrian1996) [inline backend/app/settings.py:196]: @JonasNiestroj 
This should be an env variable, so we can set it per system (staging, preprod, prod) according to our needs.

Comment (21adrian1996) [reply backend/app/settings.py:196]: And I would just call it ENABLE_REDOC independently of debugging. So we can allow debugging without redoc.
Or what would be the reason to link this to the debugging setting?

**Review by 21adrian1996 — CHANGES_REQUESTED** [review]

File: backend/app/settings.py (hunk 1 above)
Comment (JonasNiestroj) [reply backend/app/settings.py:196]: Makes sense. It was just how we implemented it in a project but lets make it more general with an env variable

Comment (JonasNiestroj) [reply backend/app/settings.py]: Switched to ENABLE_REDOC in the env file

**Review by 21adrian1996 — APPROVED** [review]

**Pull request merged**